firefox-ios/Providers/Profile.swift

```diff
@@ -603,16 +604,27 @@ open class BrowserProfile: Profile {
         if let accountManager = self.rustFxA.accountManager {
             accountManager.deviceConstellation()?.pollForCommands { commands in
                 guard let commands = try? commands.get() else { return }
-                let urls = commands.compactMap { command in
+
+                var receivedTabURLs: [URL] = []
+                var closedTabURLs: [URL] = []
+                for command in commands {
                     switch command {
                     case .tabReceived(_, let tabData):
-                        let url = tabData.entries.last?.url ?? ""
-                        return URL(string: url, invalidCharacters: false)
-                    default:
```

Comment:
Issue: Please keep the `default` case; that'll avoid breaking the iOS build when we add new commands.

(We unfortunately can't mark `IncomingDeviceCommand` as non-frozen to have the compiler enforce that, because I think the only way to mark enums as non-frozen is to build `MozillaRustComponentsSwift` with [library evolution](https://www.swift.org/blog/library-evolution/) enabled).